### DISCUSSION TITLE
Possible bug?

### DISCUSSION THREAD

**Discussion by SteveRosam** [squidfunk/mkdocs-material#5659] [General]
I have a link like this in my docs:

<https://endpoint-[YOUR_ORGANIZATION_ID].my-domain.com/index.html>

The user should sub out their org id for the text `[YOUR_ORGANIZATION_ID]`.

The build hangs when running `mkdocs build --no-directory-urls`

I get a message saying `Error reading page 'xxx.md': 'YOUR_ORGANIZATION_ID' does not appear to be an IPv4 or IPv6 address`

Removing the `<>` around the link fixed the issue. 

But it took a while to get the error because the build fails after 15-20 hours.

It builds ok locally. But fails on GH Pages/Actions

**Comment by squidfunk** [comment]
This is likely not a bug, but is definitely not a problem with Material for MkDocs. It will also be the case for all other MkDocs themes. It could be related to some extension or plugin that checks links, or MkDocs itself, but without further information and a detailed reproduction, we can only guess.
